text/web-servers/0006-web-servers-rearchitecture.md

```diff
@@ -0,0 +1,270 @@
+# Web Server Buildpack Re-architecture
+
+## Summary
+
+The Nginx and HTTPD buildpacks will be broken into three buildpacks each to
+allow for features to decrease the complexity of each of the component
+buildpacks and to allow for easier use of the underlying dependencies.
+
+## Motivation
+
+Currently Nginx and HTTPD behave as monolithic buildpacks, installing a
+dependency, running configuration, and writing a start command. This is
+antithetical to the Paketo buildpack philosophy as it means that the buildpacks
+are not modular and it is hard to use components of the buildpack easily, such
+as installing just the dependency which is an issue in buildpacks such PHP.
```

Comment:
Would it be possible to elaborate a big more on the motivation? Why is this an issue for PHP? What use cases are impossible to meet now that would be enabled by this re-architecture?

Reply:
I think I have a more solid paragraph explaining this now but let me know if it is still unclear.

Reply:
Thanks for the added context!